lib/interceptor.js

```diff
@@ -43,8 +43,12 @@ function Interceptor(scope, uri, method, requestBody, interceptorOptions) {
     this.optional = false;
 }
 
-Interceptor.prototype.optionally = function optionally() {
-    this.optional = true;
+Interceptor.prototype.optionally = function optionally(value) {
+    // The default behaviour of optionally() with no arguments is to make the mock optional.
+    value = (typeof value === 'undefined') ? true : value;
```

Comment:
There are other ways we could structure this, e.g.:

```js
if (typeof value === 'undefined') {
  this.optional = true;
} else {
  this.optional = value;
}
```

I'm happy to restyle this however you guys, the maintainers, feel is best :heart:

Reply:
I'm happy with this pattern.

Reply:
same :)